editor/styling/stylesheets/modules/_diff-view.scss

```diff
@@ -24,7 +24,7 @@
 
   .code-view {
     -fx-pref-width: 300px;
-    -fx-background-color: -df-background-lighter;
+    -fx-background-color: -df-background-light;
```

Comment:
Same here - CSS cleanup made it so that line changes were invisible in the Diff View. 😬